dev/TeachingTip/TeachingTip_rs1_themeresources.xaml

```diff
@@ -68,11 +70,88 @@
     <GridLength x:Key="TeachingTipTailMargin">10</GridLength>
 
     <x:Double x:Key="TeachingTipAlternateCloseButtonSize">40</x:Double>
-    <x:Double x:Key="TeachingTipAlternateCloseButtonGlyphSize">12</x:Double>
-
+    <x:Double x:Key="TeachingTipAlternateCloseButtonGlyphSize">16</x:Double>
+    <Thickness x:Key="TeachingTipAlternateCloseButtonPadding">4</Thickness>
+    <x:String x:Key="TeachingTipAlternateCloseButtonGlyph">&#xE711;</x:String>
+   
     <Thickness x:Key="TeachingTipContentMargin">12</Thickness>
 
     <x:Double x:Key="TeachingTipTopHighlightHeight">1</x:Double>
     <x:Double x:Key="TeachingTipBorderThickness">1</x:Double>
     <Thickness x:Key="TeachingTipTopHighlightOffsetForBorder">0,1,0,0</Thickness>
+
+    <Style x:Key="AlternateCloseButtonStyle" TargetType="Button">
+        <Setter Property="Width" Value="{ThemeResource TeachingTipAlternateCloseButtonSize}" />
+        <Setter Property="Height" Value="{ThemeResource TeachingTipAlternateCloseButtonSize}" />
+        <Setter Property="Background" Value="{ThemeResource SubtleFillColorTransparentBrush}" />
+        <Setter Property="Foreground" Value="{ThemeResource TextFillColorPrimaryBrush}" />
+        <Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />
+        <Setter Property="FontSize" Value="{ThemeResource TeachingTipAlternateCloseButtonGlyphSize}" />
+        <Setter Property="VerticalAlignment" Value="Top" />
+        <Setter Property="HorizontalAlignment" Value="Right" />
+        <Setter Property="Padding" Value="{ThemeResource TeachingTipAlternateCloseButtonPadding}"/>
+        <Setter Property="BorderThickness" Value="{ThemeResource TeachingTipAlternateCloseButtonBorderThickness}" />
+        <Setter Property="Content" Value="{ThemeResource TeachingTipAlternateCloseButtonGlyph}"/>
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
+        <Setter Property="FocusVisualMargin" Value="1"/>
+        <Setter Property="Template">
```

Comment:
Instead of overriding the template, we could locally override the light weight styles that you changed with this new template.  I'm not sure which is better.